### ISSUE TITLE
last_operation is reported as change, and can not be applied

### ISSUE THREAD

**Issue by dpittner** [IBM-Cloud/terraform-provider-ibm#3131]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* ibm_XXXXX

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please share a link to the ZIP file.
```

### Expected Behavior

No change should be reported. 
### Actual Behavior

No matter if I apply the change, the plan will report as: 

  # ibm_resource_instance.sysdig_instance will be updated in-place
  ~ resource "ibm_resource_instance" "sysdig_instance" {
        id                      = "crn:v1:bluemix:public:sysdig-monitor:us-south:a/...:...::"
      + last_operation          = (known after apply)
        name                    = "Sysdig-DAL-PROD"
      + parameters              = {}
        tags                    = []
        # (31 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

### Steps to Reproduce

Need an instance of a service created with a backlevel TF provider (e.g. march)

1. `terraform apply`

### Important Factoids

Current provider version 0.32.1

### References

* #0000

**Comment by kavya498** [comment]
@dpittner ,

There was a bug in 1.22.0 `last_operation` was marked as `optional` and hence you are seeing a diff while migrating from 1.22.0..
This is been fixed in 1.24.0 release
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/2532
Kindly, let us know if you still see this issue..
Thanks.

**Comment by kavya498** [comment]
Closing this issue due to inactivity,
Kindly open a new issue and tag this issue, if this issue still persits
Thanks..
